lightning/src/ln/channelmanager.rs

```diff
@@ -510,7 +510,11 @@ impl<'a> Drop for PersistenceNotifierGuard<'a> {
 
 /// The amount of time in blocks we require our counterparty wait to claim their money (ie time
 /// between when we, or our watchtower, must check for them having broadcast a theft transaction).
-pub(crate) const BREAKDOWN_TIMEOUT: u16 = 6 * 24;
+///
+/// This can be incrased (but not decreased) through [`ChannelHandshakeConfig::our_to_self_delay`]
+///
+/// [`ChannelHandshakeConfig::our_to_self_delay`]: `crate::util::config::ChannelHandshakeConfig::our_to_self_delay`
```

Comment:
Should there be ticks on the right-hand side? I'm going off of this which may not be current:

https://rust-lang.github.io/rfcs/1946-intra-rustdoc-links.html#additions-to-the-documentation-syntax

Reply:
Hmm, I guess its not required? https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html seems to agree with you. It definitely worked with `cargo doc` for me with backticks, however.

Reply:
Would you prefer if I drop them everywhere?

Reply:
Yeah, let's drop them as it would be simpler going forward without them. They are only on the left-hand side because they need to match the text.